lib/grape-swagger/doc_methods/entity_parser/old.rb

```diff
@@ -0,0 +1,71 @@
+module GrapeSwagger
+  module DocMethods
+    module OldEntityParser
+      def parse_entity_models(models)
```

Comment:
good idea, may be it could adopted for the swagger2.0 branch …

think functionality is implemented, now it would be refactored to have a maintable code, isn't it?
mean, please split it of into some more methods to better dive in, thanks ;)